pkg/threescale/backend/helpers.go

```diff
@@ -104,6 +104,33 @@ func validateTransactions(transactions []api.Transaction) error {
 	return nil
 }
 
+func getCounterDiffs(compareExisting, toNew LimitCounter) (added, removed []string) {
```

Comment:
Needs some comments to explain what this does, ie. compares two sets of limited counters (ie. local and remote states) and returns which metrics were added in the second argument and which were removed. I guess the func name actually confused me, because I was expecting some kind of deltas. Maybe we can find a name that better matches this behaviour? How about something along the lines of `computeAddedAndRemovedMetrics(src, dst LimitCounter) (added, removed []string)`?